dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
 console.level.info=Info, audit, and warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.audit=Audit and warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.warning=Warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.error=Error messages will be written to the system error stream.
-console.level.off=No server output will be written to system streams.  Only JVM output will be written to system streams.
+console.level.off=No server output will be written to system streams. Only JVM output will be written to system streams.
 
 console.copySystemStreams=Copy System.out and System.err to system streams
-console.copySystemStreams.desc=If true, write System.out to the system output stream and System.err to the system error stream. \
-If false, System.out and System.err will write to configured logs like messages.log or trace.log, but not to the system streams. \
-The default value is true.
+console.copySystemStreams.desc=If true, messages that are written to the System.out and System.err streams are copied to console.log. If false, those messages are written to configured logs such as messages.log or trace.log, but they are not copied to console.log. The default value is true. Avoid trouble: Use the com.ibm.ws.logging.console.copySystemStreams property, except in cases where you are trying to change the configuration dynamically after startup.
 #
 # Trace specification and format
 #
 trace.specification=Trace specification
-trace.specification.desc=A trace specification that conforms to the trace specification grammar and specifies the initial state \
-for various trace components. An empty value is allowed and treated as 'disable all trace'. Any component that is not specified \
-is initialized to a default state of *=info.
+trace.specification.desc=The trace specification is used to selectively enable trace. A trace specification that conforms to the trace specification grammar and specifies the initial state for various trace components. An empty value is allowed and treated as 'disable all trace'. Any component that is not specified is initialized to a default state of *=info.
```

Comment:
Change
"The trace specification is used to selectively enable trace. A trace specification that conforms to the trace specification grammar and specifies the initial state for various trace components."
to
"A trace specification that conforms to the trace specification grammar and specifies the initial state for various trace components. The trace specification is used to selectively enable trace."